downstream/modules/aap-on-azure/proc-azure-resource-quotas.adoc

```diff
@@ -0,0 +1,61 @@
+////
+Base the file name and the ID on the module title. For example:
+* file name: con-my-concept-module-a.adoc
+* ID: [id="con-my-concept-module-a_{context}"]
+* Title: = My concept module A
+////
+
+[id="proc-azure-resource-quotas"]
+
+= Azure resource quotas and infrastructure limits
+
+Microsoft imposes resource limits within each Azure region. The limit for CPU is the most likely to impact Red Hat Ansible Automation Platform on Microsoft Azure.
+
+Before you install Ansible Automation Platform for Azure, ensure that you have capacity to deploy the managed application into your desired region.
+Refer to xref:con-aap-azure-infrastructure-usage[Azure infrastructure usage] for infrastructure requirements.
+
+== Regional vCPU limits
+
+The Azure resources used during the deployment of the managed application temporarily exceed the resource requirements in xref:con-aap-azure-infrastructure-usage[Azure infrastructure usage]. The `Total Regional vCPUs` quota is temporarily consumed when deploying the managed application.
+
+Every Azure region has a separate `Total Regional vCPUs` quota. To prevent installation failure, ensure that you have at least 8 available vCPUs in the Azure region where you want to deploy the managed application.
+
+The following steps describe how to view the resource quotas for your subscription the Azure console:
+
+. In the Azure console, search for *Quotas* and open the *My Quotas* page.
+. Select the region where you wish to deploy the managed application to view your allocation and usage metrics for that region.
++
+Ensure that you have selected a single region. Viewing all regions at once will not show the limitations of a single Azure region.
+
+== Regional StandardCore limits
+
+The `StandardCore` limit is a compute metric for the resources that are temporarily consumed when deploying the managed application.
+
+It is possible that the Ansible Automation Platform for Azure will deploy without hitting the `StandardCore` limit.
+When a deployment fails because the consumed resources hit the `StandardCore` limit, the error message includes `container group quota 'StandardCores' exceeded`:
+
+----
+code: DeploymentFailed
+message:
+  At least one resource deployment operation failed. Please list deployment operations for details. 
+  Please see https://aka.ms/DeployOperations for usage details.
+details:
+  - code: DeploymentScriptContainerGroupInvalidSettings
+    message:
+      Resource type 'Microsoft.ContainerInstance/containerGroups' 
+      container group quota 'StandardCores' exceeded in region 'eastus'.
+      Limit: '10', Usage: '10' Requested: '1'.
+----
+
+The `StandardCore` metric is not displayed in the *My Quotas* page in the Azure console. To request the value of your regional limit, contact Microsoft directly.
+
+If your deployments fail because the consumed resources reach this limit, you must submit a resource increase request for `StandardCore` to Microsoft.
```

Comment:
The following content feels like it could be a subsection within `== Regional StandardCore limits` because you'll only need to contact Microsoft to request a limit increase. Perhaps something like `=== Requesting a StandardCore limit increase`